parsl/dataflow/task_status_poller.py

```diff
@@ -43,7 +43,7 @@ def poll(self, now: float):
         if self._should_poll(now):
             self._status = self._executor.status()
             self._last_poll_time = now
-            self.send_monitoring_info(self._status, block_id_type='internal')
+            self.send_monitoring_info(self._status, block_id_type='external')
```

Comment:
I think external and internal have been replaced by "block" and "job" so this parameter should be tidied up too: call it 'block' and 'job' perhaps

Reply:
Fixed